bootstrap_terraform.py

```diff
@@ -6,27 +6,35 @@
 import os
 import sys
 import yaml
+import json
 
 
 def main():
     if 'AWS_PROFILE' in os.environ:
         boto3.setup_default_session(profile_name=os.environ['AWS_PROFILE'])
     if 'AWS_REGION' in os.environ:
-        ssm = boto3.client('ssm', region_name=os.environ['AWS_REGION'])
+        secrets_manager = boto3.client(
+            'secretsmanager', region_name=os.environ['AWS_REGION'])
     else:
-        ssm = boto3.client('ssm')
+        secrets_manager = boto3.client('secretsmanager')
 
     try:
-        parameter = ssm.get_parameter(Name='terraform_bootstrap_config', WithDecryption=False)
+        response = secrets_manager.get_secret_value(
+            SecretId="/concourse/dataworks/terraform")
     except botocore.exceptions.ClientError as e:
         error_message = e.response["Error"]["Message"]
         if "The security token included in the request is invalid" in error_message:
```

Comment:
sorry I know this push wasn't part of your PR but we may run into issues if boto error message changes. 
although this shouldn't be an issue if we were specifying our boto version which is something that we have planned as tech debt. so it may not be an issue soon enough